src/containers/Dashboard/container.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import PropTypes from 'prop-types'
+
+import { requestData } from 'Store/data/actions'
+import { TRENDING_MOVIES, ALL_TRENDING_DAY } from 'Constants/'
+import { trendingMoviesSchema } from 'Utils/'
+import DashboardComponent from './component'
+
+class Dashboard extends Component {
+  componentDidMount() {
+    const { getMovies } = this.props
+    getMovies(ALL_TRENDING_DAY, TRENDING_MOVIES, trendingMoviesSchema)
+  }
+
+  onPageChange = (page) => {
+    const { getMovies } = this.props
+    getMovies(`${ALL_TRENDING_DAY}?page=${page}`, TRENDING_MOVIES, trendingMoviesSchema)
```

Comment:
not sure about passing api url via action. it creates some unnecessary connection between view level and business logic. possibly it would be better to pass needed variables to the action and then build api url in the operation. what about schema: is it possible to have only one `moviesSchema` for all movies endpoints? if so we can keep it inside operation too